doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_memory.rst

```diff
@@ -36,6 +34,7 @@ There are multiple RAM banks in the system.
 Each local domain (like Application or Radio) contains its own RAM.
 There is also a large part of the RAM in the global domain to be shared between the cores in the system.
 
+.. to review
 
 .. image:: images/nrf54h20_memorymap.png
```

Comment:
I am not certain if this diagram should be located in this specific chapter - Diagram represents entire nRF54H20 Memory Layout, while chapter is focused on RAM